frontend/src/views/Core/ForgotPassword/ForgotPassword.scss

```diff
@@ -0,0 +1,31 @@
+@import "src/scss/Core";
```

Comment:
In the css of each component page, _we need only the css for the page component_, for general components we need the logic on Core.scss